justfile

```diff
@@ -45,6 +57,12 @@ deps: dart-deps rust-deps install-async-bindgen
 dart-fmt:
     cd "$DART_WORKSPACE"; \
     dart format {{ if env_var_or_default("CI", "false") == "true" { "--output=none --set-exit-if-changed" } else { "" } }} .
+    cd "$FLUTTER_WORKSPACE"; \
+    dart format {{ if env_var_or_default("CI", "false") == "true" { "--output=none --set-exit-if-changed" } else { "" } }} .
+
+# Installs the required rust toolchain version
+install-rust-toolchain:
```

Comment:
This is misleading because the nightly toolchain is only needed to format while 1.55 is needed for everything else.
Also I'm not too incline to add stuff like this here since we don't have the same for dart or flutter toolchains and is something that we need to do once on our machine

Reply:
Mmh, I don't see why not having feature parity is a reason to exclude it. I added this because I was confused that we're referencing the `RUST_NIGHTLY` variable in the scripts, but don't have a way to install it. As a result, I myself was using a much more recent version of Rust nightly to format my code, which could've led to odd problems. 

In any case, I'll remove it for now, as it is probably not the right place to put it.